packages/conventional-changelog-writer/index.js

```diff
@@ -12,9 +12,11 @@ import {
 const dirname = fileURLToPath(new URL('.', import.meta.url))
 
 // sv-SE is used for yyyy-mm-dd format
-const dateFormatter = Intl.DateTimeFormat('sv-SE', {
-  timeZone: 'UTC'
-})
+function dateFormatter(timeZone) {
+  return Intl.DateTimeFormat('sv-SE', {
+    timeZone
+  })
+}
```

Comment:
Let's rewrite it to a simpler format function

```suggestion
function formatDate(date, timeZone = 'UTC') {
  // sv-SE is used for yyyy-mm-dd format
  return Intl.DateTimeFormat('sv-SE', {
    timeZone
  }).format(date)
}
```

Reply:
Thanks for the review PR.
Fixed based on your advice!